src/index.html

```diff
@@ -0,0 +1,213 @@
+                </div>
+                <div class="tweeting-container2">
+                    <div class="tag-name2">
+                        <p><strong>Amber</strong><span class="at2"> @amber ∙ 54m </span></p>
+                    </div>
+                    <div class="actual-tweet2"><p><span>@sangeet @yuvi</span> This is no time for ease and comfort. It is time to dare and endure.</p></div>
+                    <div class="actual-tweet-icon2">
+                        <div class="tweet-icon-one2">
+                            <img src="../assets/message-circle-svgrepo-com.svg" alt="">
+                        </div>
+                        <div class="tweet-icon-two2">
+                            <img src="../assets/retweet-svgrepo-com.svg" alt="">
+                        </div>
+                        <div class="tweet-icon-three2">
+                            <img src="../assets/heart-svgrepo-com.svg" alt="">
+                        </div>
+                        <div class="tweet-icon-four2">
+                            <img src="../assets/file-upload-svgrepo-com.svg" alt="">
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class="tweetThree-container">
+                <div class="user-posting-pic3">
+                    <img src="../assets/elephant.png" alt="">
+                </div>
+                <div class="tweeting-container3">
+                    <div class="tag-name3">
+                        <p><strong>Yuv</strong><span class="at3"> @YuvPur ∙ 2h </span></p>
+                    </div>
+                    <div class="actual-tweet3"><p><span>@sangeet @amber</span> Be thankful for what you have; you'll end up having more.</p></div>
+                    <div class="actual-tweet-icon3">
+                        <div class="tweet-icon-one3">
+                            <img src="../assets/message-circle-svgrepo-com.svg" alt="">
+                        </div>
+                        <div class="tweet-icon-two3">
+                            <img src="../assets/retweet-svgrepo-com.svg" alt="">
+                        </div>
+                        <div class="tweet-icon-three3">
+                            <img src="../assets/heart-svgrepo-com.svg" alt="">
+                        </div>
+                        <div class="tweet-icon-four3">
+                            <img src="../assets/file-upload-svgrepo-com.svg" alt="">
+                        </div>
+                    </div>
+                </div>
+            </div>
+
+        </div>
+
+        <!-- Right-Bar -->
+        <div class="right-bar">
+            <div class="whats-happening-right"><p><strong>What's Happening</strong></p></div>
+            <div class="who-to-follow-right"><p><strong>Who to follow</strong></p></div>
+        </div>
+
+        <!-- Footer -->
+        <div class="footer"></div>
+
+     <!-- Container Ends-->
+    </div>
+</body>
+</html>
```

Comment:
Try to use BEM metodology you read more about it in Google it is very good metodology for class names :)

Reply:
What's BEM methodology?  👀

Reply:
https://www.youtube.com/watch?v=MNPdifWAAa4

You can have a look this video about BEM it is very usefull when you will use scss or sass :)

Reply:
Thanks Nazar, that's really helpful - I'll watch this vid.